### DISCUSSION TITLE
Adding Adobe Section

### DISCUSSION THREAD

**Discussion by PJGitHub9** [blocklistproject/Lists#852] [Ideas]
Im surprised one isn't easy to find. Is there anyway to get a list added to block all Adobe requests?

They have documentation on what links they require to be open:

https://helpx.adobe.com/ca/enterprise/kb/network-endpoints.html

Theres also an existing blocklist that hasnt been maintained:  https://helpx.adobe.com/ca/enterprise/kb/network-endpoints.html

**Comment by PJGitHub9** [comment]
[Adobe Major Updated Blocklist.txt](https://github.com/PJGitHub9/PJsAdobeBlocklist)


Here's a version you can use for people to test :) it has all the major links in it!

**Comment by toni-online** [comment]
There is also this Adobe Block List available

https://github.com/Ruddernation-Designs/Adobe-URL-Block-List

**Comment by AxiomOperator** [comment]
We can bring this in, we had another request for a ai chat list. We will work on these
